packages/laconia-acceptance-test/acceptance-test/acceptance.spec.js

```diff
@@ -7,7 +7,7 @@ const laconiaTest = require("@laconia/test");
 const WebSocket = require("ws");
 
 const SERVERLESS_SERVICE_NAME = "laconia-acceptance";
-const SERVERLESS_STAGE = process.env.NODE_VERSION;
+const SERVERLESS_STAGE = `node${process.env.NODE_VERSION}`;
```

Comment:
You could add default here, if does it, package.json will require only: test, deploy and remove, use system values or default :-)

Reply:
Thanks for the input! My original intention is to make sure that user or circleci can run without setting system env, so that it's easier to run the command without reading any documentation, which is why in package.json, we have `test:10`, and `deploy:10` 🤔 

I'm not precious about this one though, once we add node 12, we can definitely revisit this.